README.md

```diff
@@ -64,7 +65,9 @@ ________________________________________________________________________________
 - Lightening fast implementation of K-Means clustering algorithm even on a single thread in native Julia.
 - Support for multi-theading implementation of K-Means clustering algorithm.
 - Kmeans++ initialization for faster and better convergence.
-- Implementation of all the variants of the K-Means algorithm.
+- Implementation of all available variants of the K-Means algorithm.
```

Comment:
Implementation of many available variants of the K-Means algorithm.

# We definitely will never implement all available variants.